conn_test.go

```diff
@@ -1110,7 +1111,7 @@ func TestServerTimeout(t *testing.T) {
 	time.Sleep(300 * time.Millisecond)
 	select {
 	case msg := <-caReadChan:
-		t.Fatalf("TestServerTimeout: Expected no additional messages from server, got: %+v", msg)
+		t.Fatalf("TestServerTimeout: Expected no additional messages from server, got: %+v, %d", msg, len(caReadChan))
```

Comment:
oops, I'll revert this line. (https://github.com/pion/dtls/pull/162 fails at this line.)